### PR TITLE
Fixing #24

### PR DESCRIPTION
**Pull request by JustinCappos**
This clarifies that package and project revocations are the same

### CODE REVIEW

File: pep-0458.txt [hunk 1]
```diff
@@ -790,10 +790,9 @@ Revoking Trust in Projects and Versions
 =======================================
 
 From time to time either a project or a version of a package will need to be revoked.
-To revoke trust in a version of a package, the bin role can simply remove the
-delegation and re-sign the bin metadata.  Similarly, an entire project may be removed
-by removing the bin metadata references to the metadata and package versions.
-All of these actions only require actions with the online bin key.
+To revoke trust in either a project or a version of a package, the bin role can
```
Comment (JustinCappos) [inline pep-0458.txt]:
```suggestion
To revoke trust in either a project or a version of a package, the associated bin-n role can 
```

File: pep-0458.txt [hunk 2]
```diff
@@ -790,10 +790,9 @@ Revoking Trust in Projects and Versions
 =======================================
 
 From time to time either a project or a version of a package will need to be revoked.
-To revoke trust in a version of a package, the bin role can simply remove the
-delegation and re-sign the bin metadata.  Similarly, an entire project may be removed
-by removing the bin metadata references to the metadata and package versions.
-All of these actions only require actions with the online bin key.
+To revoke trust in either a project or a version of a package, the bin role can 
+simply remove the delegation and re-sign the bin metadata.  This action only 
+requires actions with the online bin key.
```
Comment (JustinCappos) [inline pep-0458.txt]:
```suggestion
requires actions with the online bin-n key.
```

File: pep-0458.txt [hunk 3]
```diff
@@ -790,10 +790,9 @@ Revoking Trust in Projects and Versions
 =======================================
 
 From time to time either a project or a version of a package will need to be revoked.
-To revoke trust in a version of a package, the bin role can simply remove the
-delegation and re-sign the bin metadata.  Similarly, an entire project may be removed
-by removing the bin metadata references to the metadata and package versions.
-All of these actions only require actions with the online bin key.
+To revoke trust in either a project or a version of a package, the bin role can 
+simply remove the delegation and re-sign the bin metadata.  This action only
```
Comment (JustinCappos) [inline pep-0458.txt]:
```suggestion
simply remove the delegation and re-sign the bin-n metadata.  This action only 
```

**Comment by JustinCappos** [comment]
> Removing a delegation from _bins_ to any _bin-n_ revokes the trust in all targets that fall into that bin and would require re-signing _bins_ with its offline key. Is this really what we want here?

Thanks for the comments.  I think I've fixed it.  Can you check please?

File: pep-0458.txt [hunk 4]
```diff
@@ -790,10 +790,9 @@ Revoking Trust in Projects and Versions
 =======================================
 
 From time to time either a project or a version of a package will need to be revoked.
-To revoke trust in a version of a package, the bin role can simply remove the
-delegation and re-sign the bin metadata.  Similarly, an entire project may be removed
-by removing the bin metadata references to the metadata and package versions.
-All of these actions only require actions with the online bin key.
+To revoke trust in either a project or a version of a package, the associated bin-n role can 
+simply remove the delegation and re-sign the bin-n metadata.  This action only
```
Comment (lukpueh) [inline pep-0458.txt]: What does *bin-n* delegate to? Shouldn't it be something like...
```suggestion
simply remove the corresponding targets and re-sign the bin-n metadata.  This action only 
```

Comment (JustinCappos) [reply pep-0458.txt]: Agreed, good catch!

**Review by lukpueh — APPROVED** [review]

**Pull request merged**